index.css

```diff
@@ -740,6 +740,14 @@ input[type="email"]:focus {
 
     .shop-controls__previous-button { left: -40px; }
     .shop-controls__next-button { right: -40px; }
+
+    .gallery__image-col-1,
+    .gallery__image-col-2,
+    .primary-slider__image,
+    .catalog__surf-image,
+    .section-alpha__image,
+    .section-betha__image,
+    .section-contact__image { max-width: 100%; height: auto;}
```

Comment:
Random note: moving this into a utility class `.img-responsive` might be a good idea, rather than listing so many classes like so.